.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
```

Comment:
Was this file generated by that husky CLI?

Reply:
Yeah, I followed up a note in the file to clarify that.